homeassistant/components/tuya/strings.json

```
@@ -617,6 +617,14 @@
       "water_level": {
         "name": "Water level"
       },
+      "water_level_status": {
+        "name": "Water level",
+        "state": {
+          "level_1": "[%key:common::state::low%]",
+          "level_2": "[%key:common::state::medium%]",
+          "level_3": "Full"
```

New common state was just merged to Dev:

```suggestion
          "level_3": "[%key:common::state::full%]"
```